### Data Description

“Housing Affordability Data System” of the U.S. Department of Housing and Urban Development. [HADS]

In [1]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns
import datetime

%matplotlib inline
sns.set_style('dark')
sns.set(font_scale=1.2)

import warnings
warnings.filterwarnings('ignore')

np.random.seed(0)

pd.set_option('display.max_columns',None)
#pd.set_option('display.max_rows',None)

In [2]:
df2005 = pd.read_csv("2005.csv", low_memory=False)

In [3]:
df2005

CONTROL  AGE1  BEDRMS  PER REGION   LMED   FMR   IPOV  BUILT  \
0      '100006110249'    43       3    1    '3'  47954   680   9930   1980   
1      '100006370140'    44       4    5    '3'  47954   760  23742   1985   
2      '100006520140'    58       3    3    '3'  47954   680  15364   1985   
3      '100007130148'    22       1    1    '3'  56785   519   9974   1980   
4      '100007390148'    48       1    1    '3'  60308   600   9930   1985   
...               ...   ...     ...  ...    ...    ...   ...    ...    ...   
46848  '746730070147'    33       3    4    '4'  57432  1168  19357   2002   
46849  '747548160145'    40       4    5    '4'  57432  1397  22779   2003   
46850  '747548240145'    44       4    4    '4'  57432  1397  20009   2003   
46851  '747548320145'    -9       2   -6    '4'  58600   817     -9   2003   
46852  '747730090144'    44       4    4    '4'  58600  1420  19402   2003   

       STATUS  NUNITS  TYPE   VALUE   ZINC2  ROOMS ZADEQ  ZSMHC METRO3  \
0           1       1     1   90000   20000      8   '1'    855    '5'   
1           1       1     1  150000   71000      8   '1'   1317    '5'   
2           1       1     1  187000   64729      6   '1'   1175    '5'   
3           1      16     1      -6   27040      3   '2'    490    '1'   
4           1      32     1      -6   14000      3   '1'    330    '2'   
...       ...     ...   ...     ...     ...    ...   ...    ...    ...   
46848       1       1     1      -6  105000      6   '1'   1271    '2'   
46849       1       1     1  499000   65000      9   '1'   1995    '2'   
46850       1       1     1  425000  280400      8   '1'   2313    '2'   
46851       3      54     1      -6      -6      4  '-6'     -6    '1'   
46852       1       1     1  550000   75000     10   '1'   1720    '1'   

       STRUCTURETYPE OWNRENT     UTILITY   OTHERCOST       COST06  \
0                  1     '1'  160.166667   33.333333   791.635925   
1                  1     '1'  117.000000   62.500000  1176.393209   
2                  1     '1'  135.000000   61.750000  1439.543534   
3                  3     '2'   64.750000    0.000000   490.000000   
4                  4     '2'   60.333333    0.000000   330.000000   
...              ...     ...         ...         ...          ...   
46848              1     '2'  320.666667    0.000000  1271.000000   
46849              1     '1'  194.666667  199.750000  3710.748075   
46850              1     '1'  279.666667  231.250000  3335.447425   
46851              5     '2'   76.000000    0.000000  1169.000000   
46852              1     '1'  169.666667   91.666667  3916.608433   

            COST12       COST08      COSTMED  ASSISTED  
0      1139.176204   900.349305   791.635925        -9  
1      1755.627006  1357.582175  1176.393209        -9  
2      2161.655001  1665.425778  1439.543534        -9  
3       490.000000   490.000000   490.000000         0  
4       330.000000   330.000000   330.000000         0  
...            ...          ...          ...       ...  
46848  1271.000000  1271.000000  1271.000000         0  
46849  5637.665840  4313.503368  3710.748075        -9  
46850  4976.609850  3848.816162  3335.447425        -9  
46851  1169.000000  1169.000000  1169.000000        -9  
46852  6040.465688  4580.967974  3916.608433        -9  

[46853 rows x 27 columns]

In [4]:
df2005.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 46853 entries, 0 to 46852
Data columns (total 27 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   CONTROL        46853 non-null  object 
 1   AGE1           46853 non-null  int64  
 2   BEDRMS         46853 non-null  int64  
 3   PER            46853 non-null  int64  
 4   REGION         46853 non-null  object 
 5   LMED           46853 non-null  int64  
 6   FMR            46853 non-null  int64  
 7   IPOV           46853 non-null  int64  
 8   BUILT          46853 non-null  int64  
 9   STATUS         46853 non-null  int64  
 10  NUNITS         46853 non-null  int64  
 11  TYPE           46853 non-null  int64  
 12  VALUE          46853 non-null  int64  
 13  ZINC2          46853 non-null  int64  
 14  ROOMS          46853 non-null  int64  
 15  ZADEQ          46853 non-null  object 
 16  ZSMHC          46853 non-null  int64  
 17  METRO3         46853 non-null  object 
 18  STRUCT

The file you downloaded for the 2005 data has a total of 46853 housing units. How many housing units does the file contain when you delete all housing units that have a market value (the variable ‘VALUE’) of less than 1000$?

In [5]:
df2005mod = df2005[df2005["VALUE"] >= 1000]

In [6]:
df2005mod .shape

(30514, 27)

Similarly as question 1, how many housing units do the data files for the year 2007, 2009, 2011 and 2013 contain after you delete housing units that have a market value of less than 1000$?

In [7]:
df2007 = pd.read_csv("2007.csv")

In [8]:
df2007

CONTROL  AGE1  BEDRMS  PER REGION   LMED   FMR  BUILT  STATUS  \
0      '100003130103'    -9       3   -6    '1'  66440  1048   2006       3   
1      '100003130203'    69       3    1    '1'  66440  1048   2006       1   
2      '100006110249'    45       3    1    '3'  49575   757   1980       1   
3      '100006370140'    47       4    5    '3'  49575   847   1985       1   
4      '100007130148'    30       2    2    '3'  55474   616   1980       1   
...               ...   ...     ...  ...    ...    ...   ...    ...     ...   
42724  '746730070147'    57       3    2    '4'  59839  1278   2002       1   
42725  '747548160145'    42       4    5    '4'  59839  1517   2003       1   
42726  '747548240145'    46       4    3    '4'  59839  1517   2003       1   
42727  '747548320145'    25       2    2    '4'  59100   782   2003       1   
42728  '747730090144'    45       5    4    '4'  59100  1310   2003       1   

        VALUE  NUNITS   ZINC2  ROOMS ZADEQ  ZSMHC METRO3  STRUCTURETYPE  \
0      140000       1      -6      5  '-6'     -6    '3'              1   
1         250       1   47400      5   '1'    477    '3'              1   
2      130000       1   26000      6   '1'    798    '5'              1   
3      300000       1  174050      7   '1'   1442    '5'              1   
4          -6       8   64000      5   '1'    715    '1'              3   
...       ...     ...     ...    ...   ...    ...    ...            ...   
42724      -6       1   55800      6   '1'   1349    '2'              1   
42725  600000       1   94000     10   '1'   2135    '2'              1   
42726  489000       1  501277      9   '1'   1708    '2'              1   
42727      -6     374       0      5   '1'    498    '1'              5   
42728  670000       1   90000      9   '1'   1959    '1'              1   

      OWNRENT     UTILITY   OTHERCOST       COST06       COST12       COST08  \
0         '1'    0.000000    0.000000   930.433662  1471.051872  1099.543363   
1         '1'  296.500000   15.083333   313.244822   314.210212   313.546804   
2         '1'  131.666667   37.500000  1033.140781  1535.143405  1190.171218   
3         '1'  134.000000   75.000000  2202.786418  3361.254012  2565.164350   
4         '2'  126.666667   19.166667   715.000000   715.000000   715.000000   
...       ...         ...         ...          ...          ...          ...   
42724     '2'  349.333333    0.000000  1349.000000  1349.000000  1349.000000   
42725     '1'  375.000000  150.000000  4512.572836  6829.508023  5237.328699   
42726     '1'  305.500000  198.333333  3753.705195  5642.007372  4344.381223   
42727     '2'  131.000000    0.000000   498.000000   498.000000   498.000000   
42728     '1'  297.000000   92.916667  4842.706333  7429.950626  5652.017047   

           COSTMED  ASSISTED  
0       930.433662        -9  
1       313.244822        -9  
2      1033.140781        -9  
3      2202.786418        -9  
4       715.000000         0  
...            ...       ...  
42724  1349.000000         0  
42725  4512.572836        -9  
42726  3753.705195        -9  
42727   498.000000         0  
42728  4842.706333        -9  

[42729 rows x 25 columns]

In [9]:
df2007mod = df2007[df2007["VALUE"] >= 1000]

In [10]:
df2007mod

CONTROL  AGE1  BEDRMS  PER REGION   LMED   FMR  BUILT  STATUS  \
0      '100003130103'    -9       3   -6    '1'  66440  1048   2006       3   
2      '100006110249'    45       3    1    '3'  49575   757   1980       1   
3      '100006370140'    47       4    5    '3'  49575   847   1985       1   
6      '100007540148'    44       3    3    '3'  55474   807   1985       1   
7      '100008700141'    -9       2   -6    '4'  52415   778   1980       3   
...               ...   ...     ...  ...    ...    ...   ...    ...     ...   
42722  '746548300148'    24       4    1    '4'  59100  1359   2001       1   
42723  '746548630148'    48       3    4    '4'  59839  1278   2003       1   
42725  '747548160145'    42       4    5    '4'  59839  1517   2003       1   
42726  '747548240145'    46       4    3    '4'  59839  1517   2003       1   
42728  '747730090144'    45       5    4    '4'  59100  1310   2003       1   

        VALUE  NUNITS   ZINC2  ROOMS ZADEQ  ZSMHC METRO3  STRUCTURETYPE  \
0      140000       1      -6      5  '-6'     -6    '3'              1   
2      130000       1   26000      6   '1'    798    '5'              1   
3      300000       1  174050      7   '1'   1442    '5'              1   
6      150000       1  145000      6   '1'   1070    '1'              1   
7      257000      12      -6      4  '-6'     -6    '4'              3   
...       ...     ...     ...    ...   ...    ...    ...            ...   
42722  250000       1   60000      8   '1'   1784    '1'              1   
42723  900000       1  313000      7   '1'   1288    '2'              1   
42725  600000       1   94000     10   '1'   2135    '2'              1   
42726  489000       1  501277      9   '1'   1708    '2'              1   
42728  670000       1   90000      9   '1'   1959    '1'              1   

      OWNRENT     UTILITY   OTHERCOST       COST06        COST12       COST08  \
0         '1'    0.000000    0.000000   930.433662   1471.051872  1099.543363   
2         '1'  131.666667   37.500000  1033.140781   1535.143405  1190.171218   
3         '1'  134.000000   75.000000  2202.786418   3361.254012  2565.164350   
6         '1'  194.916667   66.666667  1258.476542   1837.710339  1439.665508   
7         '1'    0.000000  125.000000  1833.010365   2825.430937  2143.447459   
...       ...         ...         ...          ...           ...          ...   
42722     '1'  284.333333   50.000000  1995.822015   2961.211676  2297.803625   
42723     '1'  557.833333  463.333333  7002.525920  10477.928700  8089.659715   
42725     '1'  375.000000  150.000000  4512.572836   6829.508023  5237.328699   
42726     '1'  305.500000  198.333333  3753.705195   5642.007372  4344.381223   
42728     '1'  297.000000   92.916667  4842.706333   7429.950626  5652.017047   

           COSTMED  ASSISTED  
0       930.433662        -9  
2      1033.140781        -9  
3      2202.786418        -9  
6      1258.476542        -9  
7      1833.010365        -9  
...            ...       ...  
42722  1995.822015        -9  
42723  7002.525920        -9  
42725  4512.572836        -9  
42726  3753.705195        -9  
42728  4842.706333        -9  

[27785 rows x 25 columns]

In [11]:
df2007mod.shape

(27785, 25)

The assignment outlined in the video lesson asks you to compare market values of occupied versus not occupied housing units across the various years. In the data files (after deleting all housing units that have a market value of less than 1000$) how many housing units are “Occupied” versus “not occupied” across various years?

In [12]:
df2005mod

CONTROL  AGE1  BEDRMS  PER REGION   LMED   FMR   IPOV  BUILT  \
0      '100006110249'    43       3    1    '3'  47954   680   9930   1980   
1      '100006370140'    44       4    5    '3'  47954   760  23742   1985   
2      '100006520140'    58       3    3    '3'  47954   680  15364   1985   
5      '100007540148'    42       3    3    '3'  56785   788  15484   1985   
8      '100010190149'    51       3    2    '3'  47954   680  12838   1985   
...               ...   ...     ...  ...    ...    ...   ...    ...    ...   
46846  '746548550148'    34       5    4    '4'  58600  1633  19442   2002   
46847  '746548630148'    46       3    4    '4'  57432  1168  19402   2003   
46849  '747548160145'    40       4    5    '4'  57432  1397  22779   2003   
46850  '747548240145'    44       4    4    '4'  57432  1397  20009   2003   
46852  '747730090144'    44       4    4    '4'  58600  1420  19402   2003   

       STATUS  NUNITS  TYPE   VALUE   ZINC2  ROOMS ZADEQ  ZSMHC METRO3  \
0           1       1     1   90000   20000      8   '1'    855    '5'   
1           1       1     1  150000   71000      8   '1'   1317    '5'   
2           1       1     1  187000   64729      6   '1'   1175    '5'   
5           1       1     1  150000   42000      7   '1'   1023    '1'   
8           1       1     1  175000   60000      5   '1'   1481    '5'   
...       ...     ...   ...     ...     ...    ...   ...    ...    ...   
46846       1       1     1  450000  200000     10   '1'   2181    '1'   
46847       1       1     1  800000  302000      7   '1'   2908    '2'   
46849       1       1     1  499000   65000      9   '1'   1995    '2'   
46850       1       1     1  425000  280400      8   '1'   2313    '2'   
46852       1       1     1  550000   75000     10   '1'   1720    '1'   

       STRUCTURETYPE OWNRENT     UTILITY   OTHERCOST       COST06  \
0                  1     '1'  160.166667   33.333333   791.635925   
1                  1     '1'  117.000000   62.500000  1176.393209   
2                  1     '1'  135.000000   61.750000  1439.543534   
5                  1     '1'  147.166667   58.333333  1202.393209   
8                  1     '1'  114.000000   29.166667  1306.208744   
...              ...     ...         ...         ...          ...   
46846              1     '1'  171.000000  167.500000  3329.179627   
46847              1     '1'  334.333333   87.500000  5738.597114   
46849              1     '1'  194.666667  199.750000  3710.748075   
46850              1     '1'  279.666667  231.250000  3335.447425   
46852              1     '1'  169.666667   91.666667  3916.608433   

            COST12       COST08      COSTMED  ASSISTED  
0      1139.176204   900.349305   791.635925        -9  
1      1755.627006  1357.582175  1176.393209        -9  
2      2161.655001  1665.425778  1439.543534        -9  
5      1781.627006  1383.582175  1202.393209        -9  
8      1981.981507  1517.595871  1306.208744        -9  
...            ...          ...          ...       ...  
46846  5066.881018  3872.746524  3329.179627        -9  
46847  8827.844031  6704.938265  5738.597114        -9  
46849  5637.665840  4313.503368  3710.748075        -9  
46850  4976.609850  3848.816162  3335.447425        -9  
46852  6040.465688  4580.967974  3916.608433        -9  

[30514 rows x 27 columns]

In [13]:
occupied2005 = df2005mod[df2005mod['STATUS'] == 1]
occupied2005

CONTROL  AGE1  BEDRMS  PER REGION   LMED   FMR   IPOV  BUILT  \
0      '100006110249'    43       3    1    '3'  47954   680   9930   1980   
1      '100006370140'    44       4    5    '3'  47954   760  23742   1985   
2      '100006520140'    58       3    3    '3'  47954   680  15364   1985   
5      '100007540148'    42       3    3    '3'  56785   788  15484   1985   
8      '100010190149'    51       3    2    '3'  47954   680  12838   1985   
...               ...   ...     ...  ...    ...    ...   ...    ...    ...   
46846  '746548550148'    34       5    4    '4'  58600  1633  19442   2002   
46847  '746548630148'    46       3    4    '4'  57432  1168  19402   2003   
46849  '747548160145'    40       4    5    '4'  57432  1397  22779   2003   
46850  '747548240145'    44       4    4    '4'  57432  1397  20009   2003   
46852  '747730090144'    44       4    4    '4'  58600  1420  19402   2003   

       STATUS  NUNITS  TYPE   VALUE   ZINC2  ROOMS ZADEQ  ZSMHC METRO3  \
0           1       1     1   90000   20000      8   '1'    855    '5'   
1           1       1     1  150000   71000      8   '1'   1317    '5'   
2           1       1     1  187000   64729      6   '1'   1175    '5'   
5           1       1     1  150000   42000      7   '1'   1023    '1'   
8           1       1     1  175000   60000      5   '1'   1481    '5'   
...       ...     ...   ...     ...     ...    ...   ...    ...    ...   
46846       1       1     1  450000  200000     10   '1'   2181    '1'   
46847       1       1     1  800000  302000      7   '1'   2908    '2'   
46849       1       1     1  499000   65000      9   '1'   1995    '2'   
46850       1       1     1  425000  280400      8   '1'   2313    '2'   
46852       1       1     1  550000   75000     10   '1'   1720    '1'   

       STRUCTURETYPE OWNRENT     UTILITY   OTHERCOST       COST06  \
0                  1     '1'  160.166667   33.333333   791.635925   
1                  1     '1'  117.000000   62.500000  1176.393209   
2                  1     '1'  135.000000   61.750000  1439.543534   
5                  1     '1'  147.166667   58.333333  1202.393209   
8                  1     '1'  114.000000   29.166667  1306.208744   
...              ...     ...         ...         ...          ...   
46846              1     '1'  171.000000  167.500000  3329.179627   
46847              1     '1'  334.333333   87.500000  5738.597114   
46849              1     '1'  194.666667  199.750000  3710.748075   
46850              1     '1'  279.666667  231.250000  3335.447425   
46852              1     '1'  169.666667   91.666667  3916.608433   

            COST12       COST08      COSTMED  ASSISTED  
0      1139.176204   900.349305   791.635925        -9  
1      1755.627006  1357.582175  1176.393209        -9  
2      2161.655001  1665.425778  1439.543534        -9  
5      1781.627006  1383.582175  1202.393209        -9  
8      1981.981507  1517.595871  1306.208744        -9  
...            ...          ...          ...       ...  
46846  5066.881018  3872.746524  3329.179627        -9  
46847  8827.844031  6704.938265  5738.597114        -9  
46849  5637.665840  4313.503368  3710.748075        -9  
46850  4976.609850  3848.816162  3335.447425        -9  
46852  6040.465688  4580.967974  3916.608433        -9  

[29440 rows x 27 columns]

In [14]:
nonoccupied2005 = df2005mod[df2005mod['STATUS'] == 3]
nonoccupied2005

CONTROL  AGE1  BEDRMS  PER REGION   LMED   FMR  IPOV  BUILT  \
25     '100029330103'    -9       3   -6    '3'  70250  1034    -9   2005   
51     '100074570143'    -9       4   -6    '3'  48850   712    -9   1990   
54     '100092130103'    -9       3   -6    '1'  67354  1006    -9   2004   
67     '100113130103'    -9       3   -6    '2'  55986   712    -9   2005   
136    '100216130103'    -9       3   -6    '3'  55133   972    -9   2004   
...               ...   ...     ...  ...    ...    ...   ...   ...    ...   
46594  '738828490143'    -9       1   -6    '2'  67646   594    -9   2003   
46720  '741357250144'    -9       2   -6    '2'  69899   903    -9   2000   
46780  '742846100142'    -9       3   -6    '3'  53343   913    -9   2001   
46786  '743150520148'    -9       2   -6    '2'  69899   903    -9   1995   
46827  '744548420245'    -9       3   -6    '4'  57432  1168    -9   2001   

       STATUS  NUNITS  TYPE   VALUE  ZINC2  ROOMS ZADEQ  ZSMHC METRO3  \
25          3       1     1  500000     -6      6  '-6'     -6    '1'   
51          3       1     1  525000     -6      9  '-6'     -6    '1'   
54          3       1     1  130000     -6      6  '-6'     -6    '2'   
67          3       1     1  350000     -6      9  '-6'     -6    '4'   
136         3       1     1  200000     -6      6  '-6'     -6    '2'   
...       ...     ...   ...     ...    ...    ...   ...    ...    ...   
46594       3      72     1  155000     -6      3  '-6'     -6    '2'   
46720       3      12     1  235900     -6      5  '-6'     -6    '2'   
46780       3       1     1  131000     -6      7  '-6'     -6    '3'   
46786       3       6     1  235900     -6      5  '-6'     -6    '2'   
46827       3       1     1  220000     -6      5  '-6'     -6    '2'   

       STRUCTURETYPE OWNRENT     UTILITY  OTHERCOST       COST06       COST12  \
25                 1     '1'   70.833333      25.00  3418.810697  5349.590020   
51                 1     '1'   55.000000      65.00  3609.126231  5636.444521   
54                 1     '1'    0.000000       0.00   863.974114  1365.976738   
67                 1     '1'   50.000000      75.00  2451.084154  3802.629680   
136                1     '1'   41.916667     209.75  1580.857612  2353.169341   
...              ...     ...         ...        ...          ...          ...   
46594              5     '1'   91.000000     197.50  1318.622983  1917.164573   
46720              3     '1'   59.000000     155.00  1781.780720  2692.722405   
46780              1     '1'   63.333333      57.25   991.203403  1497.067585   
46786              3     '1'   59.000000     175.00  1801.780720  2712.722405   
46827              1     '1'  150.416667     105.00  1717.526707  2567.069609   

            COST08      COSTMED  ASSISTED  
25     4022.773916  3418.810697        -9  
51     4243.287612  3609.126231        -9  
54     1021.004551   863.974114        -9  
67     2873.858408  2451.084154        -9  
136    1822.442900  1580.857612        -9  
...            ...          ...       ...  
46594  1505.851581  1318.622983        -9  
46720  2066.730567  1781.780720        -9  
46780  1149.441766   991.203403        -9  
46786  2086.730567  1801.780720        -9  
46827  1983.270523  1717.526707        -9  

[1074 rows x 27 columns]

What is the mean of the market value of an occupied housing unit in year 2007 (after deleting all units with a market value less than 1000$)? Please round your answers to two decimal places.

In [15]:
occupied2007 = df2007mod[df2007mod['STATUS'] == 1]
occupied2007

CONTROL  AGE1  BEDRMS  PER REGION   LMED   FMR  BUILT  STATUS  \
2      '100006110249'    45       3    1    '3'  49575   757   1980       1   
3      '100006370140'    47       4    5    '3'  49575   847   1985       1   
6      '100007540148'    44       3    3    '3'  55474   807   1985       1   
9      '100010190149'    53       3    2    '3'  49575   757   1985       1   
10     '100013130103'    49       5    4    '2'  57757   974   2005       1   
...               ...   ...     ...  ...    ...    ...   ...    ...     ...   
42722  '746548300148'    24       4    1    '4'  59100  1359   2001       1   
42723  '746548630148'    48       3    4    '4'  59839  1278   2003       1   
42725  '747548160145'    42       4    5    '4'  59839  1517   2003       1   
42726  '747548240145'    46       4    3    '4'  59839  1517   2003       1   
42728  '747730090144'    45       5    4    '4'  59100  1310   2003       1   

        VALUE  NUNITS   ZINC2  ROOMS ZADEQ  ZSMHC METRO3  STRUCTURETYPE  \
2      130000       1   26000      6   '1'    798    '5'              1   
3      300000       1  174050      7   '1'   1442    '5'              1   
6      150000       1  145000      6   '1'   1070    '1'              1   
9      200000       1   85000      5   '1'   1081    '5'              1   
10     235000       1  172920      9   '1'   2290    '1'              1   
...       ...     ...     ...    ...   ...    ...    ...            ...   
42722  250000       1   60000      8   '1'   1784    '1'              1   
42723  900000       1  313000      7   '1'   1288    '2'              1   
42725  600000       1   94000     10   '1'   2135    '2'              1   
42726  489000       1  501277      9   '1'   1708    '2'              1   
42728  670000       1   90000      9   '1'   1959    '1'              1   

      OWNRENT     UTILITY   OTHERCOST       COST06        COST12       COST08  \
2         '1'  131.666667   37.500000  1033.140781   1535.143405  1190.171218   
3         '1'  134.000000   75.000000  2202.786418   3361.254012  2565.164350   
6         '1'  194.916667   66.666667  1258.476542   1837.710339  1439.665508   
9         '1'  156.000000   41.666667  1526.857612   2299.169341  1768.442900   
10        '1'  225.000000   73.750000  1860.549361   2768.015643  2144.412074   
...       ...         ...         ...          ...           ...          ...   
42722     '1'  284.333333   50.000000  1995.822015   2961.211676  2297.803625   
42723     '1'  557.833333  463.333333  7002.525920  10477.928700  8089.659715   
42725     '1'  375.000000  150.000000  4512.572836   6829.508023  5237.328699   
42726     '1'  305.500000  198.333333  3753.705195   5642.007372  4344.381223   
42728     '1'  297.000000   92.916667  4842.706333   7429.950626  5652.017047   

           COSTMED  ASSISTED  
2      1033.140781        -9  
3      2202.786418        -9  
6      1258.476542        -9  
9      1526.857612        -9  
10     1860.549361        -9  
...            ...       ...  
42722  1995.822015        -9  
42723  7002.525920        -9  
42725  4512.572836        -9  
42726  3753.705195        -9  
42728  4842.706333        -9  

[26466 rows x 25 columns]

In [16]:
occupied2007["VALUE"].mean()

278960.75334391295

What is the standard deviation of the market value of an occupied housing unit in year 2007 (after deleting all units with a market value less than 1000$)? Please round your answers to two decimal places.

In [17]:
occupied2007["VALUE"].std()

317162.7658762154

What is the mean and standard deviation of the market value of an not-occupied housing unit in year 2013 (after deleting all units with a market value less than 1000$)? Please round your answers to two decimal places.

In the video lesson you are asked to investigate whether there is a difference in market values of occupied versus not occupied housing units across various years. What would be an appropriate statistical test to do so?

If you were to do a difference in means hypothesis t-test, which of the following may not be an appropriate test?

What would be an appropriate null and alternate hypothesis for the a difference in means test. Please mark all that apply.

Please report the t-statistic from the difference in means test using the 2011 data (after deleting housing units with market value of less than 1000$). Please round your answer to four decimal places

Please report the t-statistic from the difference in means test using the 2013 data (after deleting housing units with market value of less than 1000$). Please round your answer to four decimal places 

In which year(s) was the market value of housing units statistically different across the occupied and unoccupied units. Please mark all years that apply.

In which year(s) was the market value of housing units statistically less for occupied units as compared to unoccupied units. Please mark all years that apply.

Based on your analysis which of the following statements are supported. Please mark all that apply.